java.js

```diff
@@ -0,0 +1,85 @@
+const openMenu = document.getElementById('hamburger');
+const closeMenu = document.querySelector('.close');
+const popUp = document.getElementById('header-nav');
+const menuList = document.querySelectorAll('.list');
+
+openMenu.addEventListener('click', () => {
+  popUp.classList.add('show');
+});
+closeMenu.addEventListener('click', () => {
+  popUp.classList.remove('show');
+});
+menuList.forEach((item) => {
+  item.addEventListener('click', () => {
+    popUp.classList.remove('show');
+  });
+});
+
+const summitSpeakers = document.getElementById('speakers');
+const people = [
+  {
+    id: 1,
+    image: './images/President-Muhammadu-Buhari.png',
+    name: 'President Muhammadu Buhari',
+    title: 'President, Federal Republic of Nigeria',
+    description: 'The president and commander in chief of the Federal Republic of Nigeria talks about the cripling, epileptic power situation in the country, and also provide suggestions as to how to stabilise the electricity situation.',
+    left: true,
+  },
+
+  {
+    id: 2,
+    image: './images/Yemi-Osinbajo.png',
+    name: 'Yemi Osibajo',
+    title: 'Vice President, Federal Republic of Nigeria',
+    description: 'The vice president gives his take on the power and electricity generation within the country, suggest plans to help the country move forward in her fight for better electrical situation.',
+  },
+
+  {
+    id: 3,
+    image: './images/Babatunde-Raji-Fashola-SAN-Governor-Lagos-State.png',
```

Comment:
- [ ] The images in this section are not being rendered because there is a typo in path, images folder should be capitalised: `./Images/Babatunde-Raji-Fashola-SAN-Governor-Lagos-State.png`. Please make sure to update the path it for all card images.
<img width="758" alt="Screenshot 2022-04-13 at 17 22 19" src="https://user-images.githubusercontent.com/77038610/163189679-15e1677d-6c1c-4c04-abf4-295bc16c5fb4.png">